setup.py

```diff
@@ -56,7 +56,7 @@
         # only needed for p2p
         "pytest-asyncio-network-simulator==0.1.0a2;python_version>='3.6'",
         # only for eth2
-        "ruamel.yaml>=0.15.87,<0.16",
+        "ruamel.yaml==0.15.98,<0.16",
```

Comment:
Probably better to remove the upper bound since it is meaningless after this change, and could be confusing:
```suggestion
        "ruamel.yaml==0.15.98",
```